typescript_sdk/packages/reality-capture/src/service/service.ts

```diff
@@ -0,0 +1,180 @@
+import axios, { AxiosInstance, AxiosResponse } from "axios";
+import { AuthorizationClient } from "@itwin/core-common";
+import { BucketResponse } from "./bucket";
+//import { DetectorsMinimalResponse, DetectorResponse } from "./detectors";
+import { CostEstimationCreate, CostEstimation } from "./estimation";
+import { Files } from "./files";
+import { Response } from "./response";
+import { JobCreate, Job, Progress, Messages, Service, getAppropriateService } from "./job";
+import { DetailedErrorResponse, DetailedError } from "./error";
+
+export class RealityCaptureService {
+    private _authorizationClient: AuthorizationClient;
+    private _axios: AxiosInstance;
+    private _serviceUrl: string;
+
+    constructor(authorizationClient: AuthorizationClient, kwargs?: { env?: string }) {
+        this._authorizationClient = authorizationClient;
+        this._axios = axios.create();
+        const env = kwargs?.env;
+        if (env === "qa") {
+            this._serviceUrl = "https://qa-api.bentley.com/";
+        } else if (env === "dev") {
+            this._serviceUrl = "https://dev-api.bentley.com/";
+        } else {
+            this._serviceUrl = "https://api.bentley.com/";
+        }
+    }
+
+    private async _getHeader(version: "v1" | "v2") {
```

Comment:
why are we supporting both v1 and v2? i would recommend pushing everyone to using the v2